### ISSUE TITLE
cSploit 2

### ISSUE THREAD

**Issue by Moto-John** [cSploit/android#896]
Hello developers!
  I've created an account just to ask, (as many others have already done...), about cSploit 2. I don't need a detailed explanation of what you are currently doing (even thought I would really like one). I only want to know if we could hope for an August release.

Sorry for asking for ETA, (if this is considered a request),and sorry if I am writing to the wrong place.
A random guy on the internet

**Comment by BearChavez** [comment]
ETA isn't possible right now, long story short

I'm not a developer, however I do wish to be one. An ETA would be hard due to the fact that developers could become inactive, slow, or just not work at all. I know eventually cSploit will want to hit the iOS scene, however when that will happen is up to the devs, just as releasing cSploit 2 will be up to devs.

**Comment by BearChavez** [comment]
I could be wrong, due to me not being a dev of even a contributor, nothing I say is set in stone

**Comment by Moto-John** [comment]
Well...I just asked because at another issue a developer said that it wouldn't take much time to assemble an alpha after he finished with the UI for which he had set a personal deadline. Here is a copy of what he said

I hereby promise a full frontend catch-up to the current (2017-06-14) state of the backend until Aug 14
After that it shouldn't take much time to assemble an alpha :)

**Comment by BearChavez** [comment]
Well, from that we can hope to see it soon then

**Comment by Moto-John** [comment]
I hope so :D

**Comment by BearChavez** [comment]
Personally, I want to help develop the app for iOS. 
I've had so many people laugh at me for using iOS for pentesting, so I want to prove it's not what they think

**Comment by Moto-John** [comment]
Well, I believe that all platforms are suitable for pentesting. The only problem is that there aren't always such tools available for every one. Let's hope that this changes

**Comment by BearChavez** [comment]
iOS has the least tools right now, so I want to change that, even if it is only a little

**Comment by Moto-John** [comment]
On the other hand we can all agree that android is a more attractive platform for pentesting than IOS. For example, on android you can install kali linux in a chroot container, install firmware for wifi dongles, install custom kernals, etc...

**Comment by BearChavez** [comment]
Android is way better, it has Kali Nethunter (IMO is better than just doing your own work on a chroot install and firmware)

**Comment by Moto-John** [comment]
Just for you to know, Nethunter is just about that. It only consists of a kali chroot, a modded kernel and some apps that everybody can install

**Comment by BearChavez** [comment]
I know, but it's easier to just flash that instead of building your own

**Comment by Silur** [comment]
Hi @Johnzoro 
The frontend catch-up is mostly done only some fancy CSS-ing is remaining, although I'd like to mention that this doesn't mean that we can release anytime, since the daemon has a long backlog too.

**Comment by FFY00** [comment]
@BearChavez your crazy if you think cSploit will be portes to iOS.

**Comment by BearChavez** [comment]
@FFY00 It's a goal for the future, ask around. I'm planning on helping them do it too

**Comment by FFY00** [comment]
@BearChavez do you have the slightest idea of how iOS works? It can be a goal but I don't think is realistic. Other thing to look is the why to port it? This is supposed to be a pentesting app, not a way for hax0r hacker wannabes to show off. But, however, that is what the community around the project appears to look like, and I'm talking about the community around, not the people driving it. So this goal makes no sense what so ever to me. I would like you to point me to where the core developers showed the slightest interest of porting this app. I can be wrong obviously, the developers can have motivations that I don't know off but I really doubt it. And, if I'm not intruding, let me ask you one thing, how do you plan to help the project if you're not an developer? I understand that you want to learn but I, in my honest opinion, don't think this is a project where you would accomplish that. This is not the right project for someone who wants to learn to program.

I know that my language usually can be somewhat aggressive but I don't mean to offend you in any way.

**Comment by BearChavez** [comment]
@FFY00 I've already learned programming but I'm unfamiliar with the whole GitHub scene, as well as iOS development. I never said I would be only aiming at porting to iOS, however in one of the Issue's I had commented saying how I do wish to assist in development and was invited by one of the devs to Slack. I may not know much about the iOS and GitHub area, but I can assure you I'm not a wannabe hacker.

I do wish to train myself in learning pentesting and so far I've been moving along with the assistance of others, me wanting to help with porting to iOS is just me messing around. There is no practicality in using an iPhone for pentesting. 

I understand it seems like I have no idea what I'm doing, and that could easily be the case, but I do wish to help this community project, I'm not a fake 10 year old hacker, and I'm not in the wrong place for what I want to do in my future

**Comment by FFY00** [comment]
@BearChavez I didn't told you were a wannabe hacker. I told that this is supposed to be an pentesting app and from and infosec point of view the work of porting the app and getting it properly working in iOS makes no sense what so ever. That work should be put instead in the current Android app and improving it. If you can tell me what are the motivations of porting the app to iOS, I would be really gladly to hear. If you want to learn more about infosec I highly suggest you to check this awesome community https://0x00sec.org (sorry if the link isn't working, I'm posting this on my phone  and I don't have much time to check the markdown documentation to confirm the syntax). If you want to join just let me know and send your email address, if you don't want to share your email here you can contact me via email (check my profile). Sorry for any typos, I really don't have much time and I'm replying on my phone.

**Comment by Silur** [comment]
@AndreaCioccarelli  my css became really ugly I'll format a patch could you please look at it before I commit and we release?

**Comment by keeganjk** [comment]
@FFY00 On the website it says it can run on any Unix platform, but currently only Androids is supported.

**Comment by FFY00** [comment]
@keeganjk the core yes, the app no. Android and iOS APIs are completely different.

**Comment by keeganjk** [comment]
@FFY00 The app can be programmed for both versions.

**Comment by pahapoika** [comment]
That's true and actually one of the goals is to get working cSploit for iOS and Android

**Comment by cobrax1** [comment]
Well i'm a starter here but i want to propose something , i used a script for dns2proxy and sslstrip2  on somesites just to test it , the site was " facebook.com " and a banc site , when the site is redirected to http facebook won't open , and the banc site descovered the use of a ssl script  and didn't wont to provide the id and password , is there any method to bypass hsts by just being men in the middle  , u recive the the cert  of site without redirecting it to http bcz that thing can be descovered

**Comment by Rinilyn** [comment]
Just another user tryna bump this topic for more juicy info about release :)
